apps/docs/content/guides/getting-started/mcp.mdx

```diff
@@ -6,211 +6,31 @@ description: 'Connect your AI tools to Supabase using MCP'
 
-- [Cursor](#cursor)
-- [Windsurf](#windsurf) (Codium)
-- [Visual Studio Code](#visual-studio-code-copilot) (Copilot)
-- [Cline](#cline) (VS Code extension)
-- [Claude desktop](#claude-desktop)
-- [Claude code](#claude-code)
-- [Amp](#amp)
+## Remote MCP installation
 
-Once connected, your AI assistants can interact with and query your Supabase projects on your behalf.
-
-## Step 1: Create an access token
-
-First, go to your [Supabase settings](/dashboard/account/tokens) and create an access token to authenticate the MCP server with your Supabase account. Give it a name that describes its purpose, like "Cursor MCP Server".
-
-## Step 2: Follow our security best practices
+### Step 1: Follow our security best practices
 
 Before running the MCP server, we recommend you read our [security best practices](#security-risks) to understand the risks of connecting an LLM to your Supabase projects and how to mitigate them.
 
-## Step 3: Configure your AI tool
-
-MCP compatible tools connect to Supabase using the [Supabase MCP server](https://github.com/supabase-community/supabase-mcp).
-
-Follow the instructions for your AI tool to connect the Supabase MCP server. The configuration below uses read-only, project-scoped mode by default. We recommend these settings to prevent the agent from making unintended changes to your database.
-
-<Admonition type="note" title="Read-only mode">
-
-Read-only mode applies only to database operations. Write operations on project-management tools,
-such as `create_project`, are still available.
-
-</Admonition>
-
-### Cursor
-
-1.  Open [Cursor](https://www.cursor.com/) and create a `.cursor` directory in your project root if it doesn't exist.
-2.  Create a `.cursor/mcp.json` file if it doesn't exist and open it.
-3.  Add the following configuration:
-
-    <$Partial path="mcp_supabase_config.mdx" variables={{ "app": "Cursor" }} />
-
-4.  Save the configuration file.
-5.  Open Cursor and navigate to **Settings > Cursor Settings > MCP & Integrations**. You should see a green active status after the server is successfully connected.
-
-### Windsurf
+### Step 2: Configure your AI tool
 
-1. Open [Windsurf](https://docs.codeium.com/windsurf) and open the Cascade assistant.
-2. Tap on the box (**Customizations**) icon, then the **Configure** icon in the top right of the panel to open the configuration file.
-3. Add the following configuration:
+Choose your project and MCP client and follow the installation instructions:
 
-   <$Partial path="mcp_supabase_config.mdx" variables={{ "app": "Windsurf" }} />
+<McpConfigPanel />
 
-4. Save the configuration file and reload by tapping **Refresh** in the Cascade assistant.
-5. You should see a green active status after the server is successfully connected.
+<Admonition type="note" title="Authenticating the server" className="mt-3">
 
-### Visual Studio Code (Copilot)
+You'll need to login with Supabase and grant certain permissions the first time you connect.
```

Comment:
```suggestion
Your MCP client will automatically prompt you to login to Supabase during setup. This will open a browser window where you can login to your Supabase account and grant access to the MCP client. Be sure to choose the organization that contains the project you wish to work with. In the future, we'll offer more fine grain control over these permissions.

Previously Supabase MCP required you to generate a personal access token (PAT), but this is no longer required.
```